### ISSUE TITLE
Please update vim/nvim documentation

### ISSUE THREAD

**Issue by zmstone** [erlang-ls/erlang-ls.github.io#26]
The current steps in vim/nvim doc did not work for me, CoC did not even try to start `erlang_ls`.
Maybe because something changed in CoC, I use `0.0.79`, while the claimed working version in [this issue](https://github.com/hyhugh/coc-erlang_ls/issues/1) was `0.0.78`,
but I am not entirely sure if that's the reason.

Anyway, I managed to get it to work by following [CoC wiki](https://github.com/neoclide/coc.nvim/wiki/Language-servers#register-custom-language-servers) which seem to be quite straightforward.
Maybe you should consider updating it with:

## Plug config for CoC in .vimrc or init.vim

`Plug 'neoclide/coc.nvim', {'branch': 'release'}`

Only this, no need for the other plugin.

## CocConfig

```
{
  "languageserver": {
    "erlang": {
      "command": "erlang_ls",
      "filetypes": ["erlang"]
    }   
  }
}
```
If `erlang_ls` is in `$PATH`, otherwise absolute path to it.

### versions
```
vim version: VIM - Vi IMproved 8.2 8021767
node version: v14.15.0
coc.nvim version: 0.0.79-be822b546f
coc.nvim directory: /home/zaiming.shi/.local/share/nvim/plugged/coc.nvim
term: dumb
platform: linux
```

### logs:
```
INFO (pid:205890) [services] - registered service "languageserver.erlang"
INFO (pid:205890) [services] - erlang state change: stopped => starting
INFO (pid:205890) [plugin] - coc.nvim 0.0.79-be822b546f initialized with node: v14.15.0 after 69ms
INFO (pid:205890) [language-client-index] - Language server "languageserver.erlang" started with 205903
INFO (pid:205890) [services] - erlang state change: starting => running
INFO (pid:205890) [services] - service languageserver.erlang started
```

**Comment by robertoaloi** [comment]
Hi! Thanks for the report. I am not a VIM user, so I can't really say. Maybe @liamjjmcnamara or @netrome could verify the suggested procedure?

**Comment by netrome** [comment]
Hi, I just tested the suggested procedure on my personal laptop and it works like a charm for me.

It seems better than depending on two separate plugins. I have the other setup with the two plugins on my work laptop, but my vague memory is that it was more complicated to set up. I am therefore positive towards updating the documentation with these instructions.

**Comment by robertoaloi** [comment]
Closed via #27
